monitoring/lifecycle/dashboard.py

```diff
@@ -66,9 +66,9 @@ def s3_request_error_rate_expr(process, job, code):
     divsLabel = 'namespace="${namespace}"'
 
     if code is not None:
-        divdLabel += f',status="{code}"'
+        divdLabel += f',status=~"{code}"'
     else:
-        divdLabel += ',status!="2xx"'
+        divdLabel += ',status!="200"'
```

Comment:
do we only ever use 200 on success?
otherwise we should have !~"2.." in such case...